app/router.js

```diff
@@ -6,7 +6,7 @@ var Router = Ember.Router.extend({
 });
 
 export default Router.map(function() {
-  this.resource('matches', function() {
+  this.route('matches', { resetNamespace: true }, function() {
```

Comment:
Why did you change this besides the `jquery` thing referenced in the description link?